QUEST_LV_0100.tsv

```diff
@@ -2001,40 +2001,40 @@ QUEST_LV_0100_20150317_001996	Try to hold it in even if it's dirty. {nl} This is
 QUEST_LV_0100_20150317_001997	You found the revelation. {nl} Aren't you hurt? {nl}
 QUEST_LV_0100_20150317_001998	Bramble. {nl} Poor thing trying to go against its fate.
 QUEST_LV_0100_20150317_001999	$The Old Man of Andale Village
-QUEST_LV_0100_20150317_002000	$So you're the one who wanted to meet Goddess Saule. {nl} Do not worry about the pond. Our priests are working on it. {nl}
-QUEST_LV_0100_20150317_002001	$The priest is located at Cerpe Crossroads. {nl}I heard he needed the Black Maize Venom, so try to get him that as a gift.
+QUEST_LV_0100_20150317_002000	$So you're the one who wants to meet Goddess Saule. {nl} If you're here about the Pond, don't worry. Our priest is working on it. {nl}
```

Comment:
Is it necessary to capitalize it? It's in a dialogue, unless it mentions a specific map name, it should. 

I've talked with vtange about this. That in dialogue with an NPC, you should use lowercase unless it is really talking about a monster (name), NPC (name), map (name) or item (name). It's more natural to use lowercase in this case.